src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1479,11 +1501,17 @@ private static FileChange GetFileChangeDetails(Range changeRange, string insertS
                 return cancelTask.Task;
             }
 
+            // If filesToAnalzye is empty, nothing to do so return early.
+            if (filesToAnalyze.Length == 0)
+            {
+                return Task.FromResult(true);
```

Comment:
I'm wondering if we want to start switching to:

`Task.CompletedTask`

https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.completedtask?redirectedfrom=MSDN&view=netframework-4.7.2

Not sure if we can but wanted to call it out.

Reply:
That is a bit clear with regard to "intention".  This property doesn't go back to NET 4.5.1 though.  Do we care about that version of .NET?  It was intro'd in 4.6 according to these docs - https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.completedtask?view=netframework-4.7.2#System_Threading_Tasks_Task_CompletedTask

Reply:
gah... you're totally right. We still care about 4.5.1 because of v3/v4.

Once we cross 2.0 we can switch to using `CompletedTask`